test/create_db.sh

```diff
@@ -5,7 +5,7 @@ source test/db-common.sh
 
 # set db connection for if running in a separate container or not
 dbconn="-u root"
-if [[ ! -z "$MYSQL_CONTAINER" ]]; then
+if [[ $MYSQL_CONTAINER ]]; then
```

Comment:
bash tokenizes the text inside [[ ]] before expanding variables so it is always safe to use naked variables and check for non-empty values using [[ $x ]] without fear that they expands into empty string or test operation:
```
$  a=""; [[ $a ]] || echo 1
1
$  a="-z"; [[ $a ]] && echo 2
2
```

As I used that form in other places, I changed that for consistency here.